pkg/provider/aws/plugin.go

```diff
@@ -63,6 +71,26 @@ func (a *AWS) Translate(result *core.ConstructGraph, dag *core.ResourceGraph) (l
 	return
 }
 
+// shouldCreateNetwork determines whether any of our aws resources will need to be within a VPC
+func (a *AWS) shouldCreateNetwork(result *core.ConstructGraph) (bool, error) {
+	constructIds, err := result.TopologicalSort()
+	if err != nil {
+		return false, err
+	}
+	for _, id := range constructIds {
```

Comment:
Does this need to be in topological order or could you just use ListConstructs (or equiv.)? If not, that'd get rid of needing `error` too.

Reply:
youre right, can just list constructs here, let me change that